ttnn/cpp/ttnn/operations/data_movement/repeat_interleave/repeat_interleave.hpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/tensor/types.hpp"
+#include "ttnn/deprecated/tt_dnn/op_library/concat/concat_op.hpp"
+#include "ttnn/deprecated/tt_dnn/op_library/repeat/repeat_op.hpp"
+#include "ttnn/operations/core/core.hpp"
+#include "ttnn/operations/data_movement/permute/permute.hpp"
+
+#include <ranges>
+
+namespace ttnn {
+namespace operations {
+namespace data_movement {
+
+namespace utils {
+    Tensor repeat_interleave(const Tensor&, uint32_t, int32_t, const std::optional<MemoryConfig>&);
+}
```

Comment:
this can be removed

Reply:
removed additional structs and namespaces